src/components/Button.js

```diff
@@ -131,34 +132,47 @@ const Button = React.createClass({
         color: this.props.primaryColor,
         fill: this.props.primaryColor,
         transition: 'all .2s ease-in',
-
+        border: '1px solid transparent',
         ':hover': {
           color: StyleConstants.adjustColor(this.props.primaryColor, -8),
           fill: StyleConstants.adjustColor(this.props.primaryColor, -8),
-          transition: 'all .2s ease-in'
+          transition: 'all .2s ease-in',
+          border: '1px solid' + StyleConstants.Colors.FOG
         },
         ':active': {
           color: StyleConstants.adjustColor(this.props.primaryColor, -16),
           fill: StyleConstants.adjustColor(this.props.primaryColor, -16),
-          transition: 'all .2s ease-in'
+          transition: 'all .2s ease-in',
+          backgroundColor: StyleConstants.Colors.PORCELAIN
+        }
+      },
+      neutral: {
+        backgroundColor: 'transparent',
+        border: '1px solid' + StyleConstants.Colors.FOG,
+        color: this.props.primaryColor,
+        ':hover': {
+          backgroundColor: StyleConstants.Colors.PORCELAIN
+        },
+        ':active': {
+          backgroundColor: StyleConstants.adjustColor(StyleConstants.Colors.PORCELAIN, -15)
         }
       },
       disabled: {
-        backgroundColor: StyleConstants.Colors.PORCELAIN,
-        borderColor: StyleConstants.Colors.PORCELAIN,
-        color: StyleConstants.Colors.FOG,
-        fill: StyleConstants.Colors.FOG
+        backgroundColor: 'transparent',
+        borderColor: StyleConstants.Colors.FOG,
+        color: StyleConstants.Colors.FOG
       },
       icon: {
-        marginTop: -6,
+        marginTop: -3,
         marginBottom: -5,
         marginLeft: -5,
-        marginRight: this.props.children ? 5 : -5
+        marginRight: this.props.children ? 5 : -5,
+        verticalAlign: 'initial'
       },
       spinner: {
-        verticalAlign: 'initial',
         marginRight: -5,
-        padding: !this.props.actionText && this.props.children ? 3 : 0
+        marginTop: -6,
+        padding: !this.props.actionText ? 3 : 0
```

Comment:
It's minor, but this would read better switched around: `this.props.actionText ? 0 : 3`